src/resource-files.ts

```diff
@@ -40,7 +41,7 @@ export const fetchResources = async function (): Promise<Resource[]> {
   if (chartDir) {
     const debug = core.getBooleanInput("debug", { required: false });
     const values = valueFiles.map((f) => ["-f", f]).flat();
-    const output = await exec.getExecOutput("helm", ["template", chartDir, ...values], { silent: !debug });
+    const output = await exec.getExecOutput("helm", ["template", chartDir, apiVersions, ...values], { silent: !debug });
```

Comment:
The arguments of the template command are described as the following:
```
Usage:
  helm template [NAME] [CHART] [flags]

Flags:
  -a, --api-versions strings                       Kubernetes api versions used for Capabilities.APIVersions
...
```

The api-versions are a flag and not a normal argument like "NAME", but you don't add the flag-name in addition to the new value if it is set. So it only would work if you add the flag-name in the YAML too, but this should be handled by the code.